src/views/Player.vue

```diff
@@ -243,13 +286,65 @@ export default {
       // If first question, go to the start of the video
       // else go to the question which came just before the current ones
 
+      this.questionJourney.push(
```

Comment:
This code is repeated a lot.
Why not have a method

```js
updateQuestionJourney(string, index) {
    this.questionJourney.push(String(this.getCurrentDateTime(), string, "q-" + index)
} 
```

Advantage of this will be that if we change the format of what we are pushing, code only needs to change in one place

In fact: looking below, all journeys can be updated from same method by adding one more parameter

Reply:
Done